Xamarin.Forms.Core/BindableLayout.cs

```diff
@@ -223,9 +223,9 @@ void ItemsSourceCollectionChanged(object sender, NotifyCollectionChangedEventArg
 					{
 						if (e.OldStartingIndex == -1)
 							goto case NotifyCollectionChangedAction.Reset;
-						for (int i = 0; i < e.OldItems.Count; ++i)
+						for (int i = 0; i < e.OldItems.Count; i++)
```

Comment:
Why change the increment operator order? This is not necessary for the changes you're doing

Reply:
not necessary, but we usually `i++` in `for` loops all over the place, so for constancy, I would not reject this change

Reply:
I see. I prefer `++i` because in some cases I think it can be a tiny bit more performant. For using it in `for` loops like this, it makes no difference.